src/core/scene/a-scene.js

```diff
@@ -149,7 +149,7 @@ module.exports = registerElement('a-scene', {
     exitVR: {
       value: function () {
         var self = this;
-        if (!this.is('vr-mode')) { return; }
+        if (!this.is('vr-mode')) { return Promise.resolve(); }
```

Comment:
I'd probably `return Promise.resolve(false);` or `return Promise.resolve('Already in VR mode.');`. either way, this fixes it so thanks 👍